systemtests/src/test/java/io/enmasse/systemtest/web/WebConsoleTest.java

```diff
@@ -223,6 +219,46 @@ public void doTestFilterConnectionsByEncrypted() throws Exception {
         receivers.forEach(AbstractClient::stop);
     }
 
+    public void doTestFilterConnectionsByUser() throws Exception {
```

Comment:
Where you stop clients? I see stopClients only in catch.

Reply:
of course it is my fault, it must be in 'finally' block (thanks again) -> fixed